client/src/components/App.jsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
 import $ from 'jquery';
-import FiveBooks from './components/FiveBooks.jsx';
+import FiveBooks from './FiveBooks.jsx';
 
 class App extends React.Component {
```

Comment:
`Airbnb` style guide suggests sticking to arrow functions for stateless/presentatiional components.